assets/properties-panel.css

```
@@ -355,6 +356,10 @@
   background-color: var(--dot-color);
 }
 
+.bio-properties-panel-dot.error {
```

`.error` sounds super generic, and I am afraid that this can cause clashes in embedded scenarios. Imagine somebody uses the class name in their project's stylesheet (e.g. `.error { background: red }`). Then, it interferes with the properties panel unintentionally.
I'd rather try to use specific class names. This can be achieved via BEM (https://getbem.com/) which we sometimes use in our projects. So in this case we'd use:

```suggestion
.bio-properties-panel-dot--error {
```